plier_recount3.smk

```diff
@@ -0,0 +1,97 @@
+"""
+PLIER recount3
+"""
+
+# ==============================================================================
+# IMPORT LIBS
+# ==============================================================================
+
+from os.path import basename
+from glob import glob
+from types import SimpleNamespace
+import os
+import utils
+import pandas as pd
+
+
+# ==============================================================================
+# SETUP
+# ==============================================================================
+
+configfile: 'conf/config.yaml'
+
+# This allows for dot (.) access instead of dict access
+# of configuration parameters.
+# EXAMPLE: config['ABC'] becomes config.ABC
+
+config = SimpleNamespace(**config)
+
+# ==============================================================================
+# RULES
+# ==============================================================================
+rule All:
```

Comment:
I have never used snakemake before, so sorry if this question does not make sense. Do you really need this "All" rule? Looks like you need to update it each time you add a new step, which is a bit annoying. Can you run snakemake with the instruction to run all the defined rules?